tests/digital_identity_service/extension/extension.builder.spec.js

```diff
@@ -0,0 +1,21 @@
+const { ExtensionBuilder } = require('../../..');
+const Extension = require('../../../src/dynamic_sharing_service/extension/extension');
```

Comment:
There a few files in the test referring to the `dynamic_sharing_service` version.
Also, I think some of these extensions are not created (copied) in the digital identity version.